lib/administrate/field/time.rb

```diff
@@ -0,0 +1,11 @@
+require_relative "base"
+
+module Administrate
+  module Field
+    class Time < Base
+      def default
+        data && data.strftime("%H:%M:%S")
```

Comment:
@jcmorrow that sounds good, like adding:

```
  time:
    formats:
      time: "%H:%M:%S"
```

to the [administrate locale file](https://github.com/thoughtbot/administrate/blob/master/config/locales/administrate.en.yml) right?